xcode-compat/build.gradle.kts

```diff
@@ -19,17 +19,18 @@ gradlePlugin {
 }
 
 dependencies {
-    implementation("org.jetbrains.kotlin", "kotlin-gradle-plugin", "1.3.20")
+    implementation("org.jetbrains.kotlin", "kotlin-gradle-plugin", "1.3.60+")
```

Comment:
May be replace the `implementation` dependency with `compileOnly`?

 The `implementation` dependency here means that this plugin will bring this `kotlin-gradle-plugin` into classpath of user's buildscript. This may cause conflicts if a user uses another version of `kotlin-gradle-plugin`. Also if a EAP version is used here, Gradle at user side can even fail to resolve this dependency if user didn't add `kotlin-eap` to repositories.

Reply:
Fixed.